File: src/main/java/no/novari/flyt/instance/gateway/model/vigo/Kontaktinformasjon.java
package no.novari.flyt.instance.gateway.model.vigo;

import lombok.Builder;
import lombok.EqualsAndHashCode;
import lombok.Getter;
import lombok.extern.jackson.Jacksonized;

@Getter
@Jacksonized
@EqualsAndHashCode
@Builder
public class Kontaktinformasjon {

    private String epostadresse;
    private String telefonnummer;
}

File: src/main/kotlin/no/fintlabs/instance/gateway/model/vigo/Kontaktinformasjon.kt
package no.fintlabs.instance.gateway.model.vigo

import com.fasterxml.jackson.annotation.JsonInclude

@JsonInclude(JsonInclude.Include.NON_NULL)
data class Kontaktinformasjon(
    val epostadresse: String? = null,
    val telefonnummer: String? = null
)
